### PR TITLE
Fixed Typographical Error in guide/english/accessibility/index.md

### PR DESCRIPTION
**Pull request by iamrajiv**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/accessibility/index.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ title: Accessibility
...
+Web accessibility also **benefits** people *without* disabilities. For example, a key principle of Web accessibility is designing Web sites and software that are flexible to meet different user needs, preferences, and situations. This **flexibility** also benefits people *without* disabilities in certain situations, such as people using a slow Internet connection, people with "temporary disabilities" such as a broken arm, and older people with changing abilities due to ageing. The article [Developing a Web Accessibility Business Case for Your Organization](https://www.w3.org/WAI/bcase/Overview) describes many different benefits of Web accessibility, including **benefits for organizations**.
```
Comment (thecodingaviator) [inline guide/english/accessibility/index.md]: We use the American version of English for this project so `aging` is correct, please revert the change

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Review by thecodingaviator — APPROVED** [review]
LGTM! 👍

**Comment by iamrajiv** [comment]
@RandellDawson is this PR good to be merged?

**Comment by RandellDawson** [comment]
@iamrajiv This has been reviewed by @thecodingaviator, so he can make that decision.

**Comment by thecodingaviator** [comment]
@iamrajiv Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**